src/hooks/useChainId.js

```diff
@@ -0,0 +1,7 @@
+import env from '../environment'
+
+function useChainId() {
+  return env('CHAIN_ID')
+}
```

Comment:
Here I was simply thinking ahead for run-time config. If we encourage using this hook it should be easier to update via state in the future.